sdk/core/Azure.Core.Experimental/src/DynamicData/BinaryDataExtensions.cs

```diff
@@ -16,8 +16,18 @@ public static class BinaryDataExtensions
         /// </summary>
         public static dynamic ToDynamicFromJson(this BinaryData utf8Json)
         {
-            MutableJsonDocument mdoc = MutableJsonDocument.Parse(utf8Json, DynamicData.DefaultSerializerOptions);
-            return new DynamicData(mdoc.RootElement);
+            DynamicDataOptions options = DynamicDataOptions.Default;
+            MutableJsonDocument mdoc = MutableJsonDocument.Parse(utf8Json, DynamicData.GetSerializerOptions(options));
+            return new DynamicData(mdoc.RootElement, options);
```

Comment:
nit: would it make sense to implement this as a call to the other overload rather than re-implementing it here?

Reply:
It is a good idea! :)